components/character_icon/commons/character_icon.lua

```diff
@@ -76,6 +78,7 @@ function CharacterIcon.Icon(args)
 	assert(iconInfo.file, 'Character:"' .. args.character .. '" has no file set')
 
 	return CharacterIcon._makeImage(iconInfo, args.size, args.class)
+		.. (args.addTextLink and ('&nbsp;' .. Page.makeInternalLink(args.character, iconInfo.link or args.character)) or '')
```

Comment:
imo use Logic.readBool here
atm the function can be used from wiki code and if you use `|addTextLink=false` (for whatever reason) it would still add the text

Reply:
```suggestion
		.. (Logic.readBool(args.addTextLink) and ('&nbsp;' .. Page.makeInternalLink(args.character, iconInfo.link or args.character)) or '')
```